File: BackEnd-ProjectAPI/Sample/Sample.Infrastructure/EntityFramework/ExpenseCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sample.Infrastructure.EntityFramework
{
    public partial class ExpenseCategory
    {
        public int Id { get; set; }
        public string Code { get; set; } = null!;
        public string Name { get; set; } = null!;
        public string? Description { get; set; }
        public bool IsActive { get; set; } = true;
        public DateTime CreatedDate { get; set; }
        public DateTime? ModifiedDate { get; set; }

        public virtual ICollection<Expense> Expenses { get; set; } = new HashSet<Expense>();
    }
}

File: FrontEnd-ProjectReact/my-react-app-inventory/src/types/expenseCategory.ts
import type { Expense } from "./expense";

export interface ExpenseCategory {
  id: number;
  code: string;
  name: string;
  description?: string;
  isActive: boolean;
  createdDate: Date;
  modifiedDate?: Date;

  // Collection of Expenses
  expenses: Expense[];
}